data/assets/scene/solarsystem/missions/jwst/timelapse.asset

```diff
@@ -1,400 +1,591 @@
+      openspace.time.setTime("]] .. dataStartTime .. [[")
+      ]] .. createForwardTimelapse() .. [[
+      openspace.time.setDeltaTime(1)
+    ]],
+    Documentation = "Jump to the JWST launch time and play the timelapse of deployment forward",
+    GuiPath = "/JWST",
+    IsLocal = false
+  }
+
+  local playBackwards = {
+    Identifier = "jwst.play.backwards",
+    Name = "Play JWST from end",
+    Command = [[
+      openspace.scriptScheduler.clear(0)
+      openspace.time.setDeltaTime(-1)
+      openspace.setPropertyValueSingle("Dashboard.JWSTStateText.Text", "")
+      openspace.time.setTime("2022 JAN 09 07:20:01")
+      ]] .. createBackwardTimelapse() .. [[
+      openspace.time.setDeltaTime(-1)
+    ]],
+    Documentation = "Jump to the end of JWST deployment time and play the timelapse of deployment in reverse",
+    GuiPath = "/JWST",
+    IsLocal = false
+  }
+
+  local clearPlay = {
+    Identifier = "jwst.play.clear",
+    Name = "Clear JWST timelapse",
+    Command = [[
+      openspace.scriptScheduler.clear(0)
+      openspace.setPropertyValueSingle("Dashboard.JWSTStateText.Text", "")
+      openspace.setPropertyValueSingle("Scene.Earth.Renderable.Layers.NightLayers.Earth_at_Night_2012.Settings.Gamma", 1.0)
+      openspace.setPropertyValueSingle("Scene.EarthAtmosphere.Renderable.Enabled", true)
+      local deltaTime = openspace.time.deltaTime()
+      if deltaTime > 0 then
+        openspace.time.setDeltaTime(1)
+      else
+        openspace.time.setDeltaTime(-1)
+      end
+    ]],
+    Documentation = "Set delta time back to realtime and clear the JWST deployment timelapse",
+    GuiPath = "/JWST",
+    IsLocal = false
+  }
+
+  local togglePlayDirection = {
+    Identifier = "jwst.toggle.direction",
+    Name = "Toggle forwards / backwards",
+    Command = [[
+      -- Flip deltatime
+      local deltaTime = openspace.time.deltaTime()
+      openspace.time.setDeltaTime(-deltaTime)
+
+      -- Switch timelapse script
+      if (deltaTime < 0) then
+        -- Going backwards and switching to forwards
+        openspace.scriptScheduler.clear(0)
+        ]] .. createForwardTimelapse() .. [[
+      else
+        -- Going forwards and switching to backwards
+        openspace.scriptScheduler.clear(0)
+        ]] .. createBackwardTimelapse() .. [[
+      end
 
-local togglePlayDirection = {
-  Identifier = "jwst.toggle.direction",
-  Name = "Toggle forwards/ backwards",
-  Command = [[
-    -- Flip deltatime
-    local deltaTime = openspace.time.deltaTime();
-    openspace.time.setDeltaTime(-deltaTime);
-
-    -- Update the dashboard text
-    local text = openspace.getPropertyValue("Dashboard.JWSTStateText.Text");
-    if(string.len(text) > 14) then
-      local newText = ""
-      if(text:sub(13, 13) == "-") then
-        newText = text:sub(1,12) .. text:sub(14)
+      -- Update the dashboard text
+      local text = openspace.getPropertyValue("Dashboard.JWSTStateText.Text")
+      if(string.len(text) > 14) then
```

Comment:
No need for the ( ) surrounding the `string.len(text) > 14`  the `if`/`then` do that job in Lua